yaki_backend/src/__test__/team-service.test.ts

```diff
@@ -18,17 +18,21 @@ describe("teamService", () => {
     const mockTeam = [
       {
         teamId: 2,
-        teamCaptainId: 2,
         teamName: "Equipe 2",
+        teamCaptainId: 2,
+        teamCustomerId: 2,
+        teamActifFlag: true,
       },
     ];
 
     it("should return a team", async () => {
       jest.spyOn(teamRepository, "getTeamByTeamMateId").mockResolvedValueOnce([
         {
           teamId: 2,
-          teamCaptainId: 2,
           teamName: "Equipe 2",
+          teamCaptainId: 2,
```

Comment:
supposed to be a list of captain as it's a many to many relation or the translation is done after ?

Reply:
ticket creation this test wasn't review during this change, serveral others backend function will need to changed given the new workflow